### PR TITLE
DUOS-1223[risk=no]Confusing options on chair vote page when no LC present

### PR DESCRIPTION
**Pull request by raejohanek**
SCOPE:
- update error message to explain what the user can do
- fix syntax error in ajax call
- replace star imports

ADDRESSES:
https://broadworkbench.atlassian.net/browse/DUOS-1223

Have you read [Terra's Contributing Guide](https://github.com/DataBiosphere/terra-ui/wiki/Contributor-Guide) lately? If not, do that first.

I, the developer opening this PR, do solemnly pinky swear that:

- [ ] PR is labeled with a Jira ticket number and includes a link to the ticket
- [ ] PR is labeled with a security risk modifier [no, low, medium, high] 
- [ ] PR describes scope of changes

In all cases:

- [ ] Get a minimum of one thumbs worth of review, preferably 2 if enough team members are available
- [ ] Get PO sign-off for all non-trivial UI or workflow changes
- [ ] Verify all tests go green
- [ ] Squash and merge; you can delete your branch after this
- [ ] Test this change deployed correctly and works on dev environment after deployment

### CODE REVIEW

**Comment by JVThomas** [comment]
@rushtong @solideoglori Quick question: [DUOS-1106](https://github.com/DataBiosphere/duos-ui/pull/974) was implemented specifically in that manner because Chairs were allowed to reject a DAR if the application itself was fundamentally un-approvable (an act that does not require a library card to be present), so why the switch to a full disable?

**Comment by solideoglori** [comment]
@JVThomas we do not want to move to a full disable

**Comment by rushtong** [comment]
Ok, I misunderstood the original requirements. As it stands now, it's confusing as some combinations allow a submit, while others don't while the missing requirements are not spelled out.

File: src/pages/access_review/VoteAsChair.js [hunk 1]
```diff
@@ -62,8 +62,9 @@ export const VoteAsChair = hh(class VoteAsChair extends React.PureComponent {
   };
 
   render() {
-    const { vote, rpVote, onUpdate, matchData, accessElectionOpen, rpElectionOpen, hasLibraryCard } = this.props;
-    const errorMessage = "The Researcher must have a Library Card before this DAR can be approved.";
+    const { vote, rpVote, onUpdate, matchData, accessElectionOpen, rpElectionOpen, libraryCards } = this.props;
+    const hasLibraryCard = !isNil(head(libraryCards));
+    const errorMessage = "The Researcher must have a Library Card before this DAR can be approved. You can still deny the request or vote on question 2.";
```
Comment (JVThomas) [inline src/pages/access_review/VoteAsChair.js]: Minor suggestion, and then the code is good to go.

With your follow up sentence I realized we can make that old first sentence a bit more specific. Something like ```The Researcher must have a Library Card before data access can be granted. You can still deny this request and/or vote on question 2```.

This way the statements describe what you can or can't do for each question

**Review by JVThomas — CHANGES_REQUESTED** [review]

**Review by JVThomas — APPROVED** [review]
Alright, PR looks good to go.

**Pull request merged**